threadit-api/Controllers/v1/ThreadController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Net.Cache;
+using ThreaditAPI.Extensions;
+using ThreaditAPI.Middleware;
+using Microsoft.AspNetCore.Mvc;
+using ThreaditAPI.Models;
+using ThreaditAPI.Models.Requests;
+using ThreaditAPI.Services;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace ThreaditAPI.Controllers.v1 {
+    [ApiController]
+    [Route("v1/thread")]
+    public class ThreadController : ControllerBase {
+        [AuthenticationRequired]
+        public async Task<IActionResult> PostThread([FromBody] PostThreadRequest request, [FromServices] ThreadService threadService) {            
```

Comment:
please add `[HttpPost()]` attribute and give it a unique endpoint eg. `[HttpPost("create")]`